lms/static/js/spec/dashboard/track_events_spec.js

```diff
@@ -14,70 +14,39 @@
                 window.edx.dashboard.trackEvents();
             });
 
-            it('sends an analytics event when the user clicks course title link', function() {
-                // Verify that analytics events fire when the 'course title link' is clicked.
-                expect(window.analytics.trackLink).toHaveBeenCalledWith(
-                    $('.course-title > a'),
-                    'edx.bi.dashboard.course_title.clicked',
-                     window.edx.dashboard.generateTrackProperties
-                );
-            });
+            it('sends analytics events when the user interacts with dashboard', function() {
+                expect(window.analytics.trackLink.callCount).toBe(7);
 
-            it('sends an analytics event when the user clicks course image link', function() {
-                // Verify that analytics events fire when the 'course image link' is clicked.
-                expect(window.analytics.trackLink).toHaveBeenCalledWith(
-                    $('.cover'),
+                var expectedArray = [
```

Comment:
can you add comments to explain each array element in terms of the tests that this replaces? Or, better yet, name each element in the array with a descriptive name, then use those to create the expected array. Also, a better name for `expectedArray` would be something like `expectedArgsForTrackLinkCall`